client/app/lib/changeteam/index.coffee

```diff
@@ -29,6 +29,6 @@ module.exports = class ChangeTeamView extends kd.ModalView
 
     controller = new ChangeTeamController()
     @addSubView list = controller.getView()
-    controller.on 'ItemsLoaded', ->
+    controller.on 'AllItemsAddedToList', ->
```

Comment:
@hakankaradis yes, you're right it was emiting that before I removed it in here https://github.com/koding/koding/pull/11150/files#diff-28c8fbaa26775fa401a9d6974d2f1232L122 ✌️ but I didn't understand how the test you provided passed in my PR when I delete that. @szkl, @usirin any ideas?